tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "es2015",
+    "jsx": "react-native",
+    "noEmit": true,
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "node",
+    "lib": ["es2015"]
+  },
+  "files": ["react-native-reanimated-tests.tsx", "react-native-reanimated.d.ts"]
```

Comment:
I wonder if it's a good place for the ts tests, it requires adding more things to the devDependencies.

Maybe we could include it in the test project at some point?

/cc @Szymon20000

Reply:
What's wrong with adding more to the devDeps? I

Reply:
Actually nothing, but seems strange to have RNGH as devDep.

I guess we can leave it in this shape for now.